types/stripe/index.d.ts

```diff
@@ -7590,14 +7591,9 @@ declare namespace Stripe {
 
         interface IPaymentIntentListOptions extends IListOptionsCreated {
             /**
-             * Filter links by their expiration status. By default, all links are returned.
-             */
-            expired?: boolean;
-
-            /**
-             * Only return links for the given file.
+             * Only return PaymentIntents for the customer specified by this customer ID.
              */
-            file?: boolean;
+            customer?: string
```

Comment:
This removed semicolon caused Travis CI to fail:
https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/628619906#L307